ui/src/components/pages/Pipeline/partials/Graph/partials/NodeInfo/partials/Pods/partials/PodDetails/partials/Metrics/index.tsx

```diff
@@ -77,7 +77,7 @@ export function Metrics({ namespaceId, pipelineId, type, vertexId }: MetricsProp
               aria-controls={`${metric?.metric_name}-content`}
               id={`${metric?.metric_name}-header`}
             >
-              <Box sx={{ textTransform: "capitalize" }}>
+              <Box>
```

Comment:
This is done so that metadata like (**in microseconds**) for the metric name doesn't get capitalized.